shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2057,6 +2057,17 @@ void WebContents::DidFinishNavigation(
       if (is_main_frame) {
         Emit("did-navigate", url, http_response_code, http_status_text);
       }
+
+      content::NavigationEntry* entry = navigation_handle->GetNavigationEntry();
+
+      // This check is needed due to an issue in Chromium
+      // Check the Chromium issue to keep updated:
+      // https://bugs.chromium.org/p/chromium/issues/detail?id=1178663
```

Comment:
FYI, The issue referenced is closed, last time I checked upstream is open for patch but no updates will happen unless we pursue it :) Maybe a tracking issue on our end is needed ?

Reply:
Thanks for the heads up. Just to confirm, do you mean making an issue in our own tracker to indicate that a patch should be made upstream?

Reply:
Yup if we want to pursue, but given the workaround is solid for years there is no real urgency as well. Just thought of highlighting since the code touched the comment, not really blocking this PR :)

Reply:
Gotcha! Yeah, I can go ahead and make an issue after this PR is merged that brings together all the relevant links/info for that as a discussion